File: zero/Maruko.Zero/Internal/Service/User/DTO/SysUserDTO.cs
using Maruko.Core.Application.Servers.Dto;
using Maruko.Core.AutoMapper.AutoMapper;

namespace Maruko.Zero
{
    [AutoMap(typeof(SysUser))]
    public class SysUserDTO : EntityDto
    {
        public long RoleId { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public string RoleName { get; set; }
        public string Icon { get; set; }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/application/services/sysUser/dto/SysUserDTO.java
package com.act.modules.zero.application.services.sysUser.dto;

import com.act.core.application.EntityDto;
import lombok.Data;

@Data
public class SysUserDTO extends EntityDto {
    //角色Id
    private Long roleId;

    //用户名
    private String userName;

    //密码
    private String password;

    //头像
    private String icon;
}
